docs/customizing_deis/choosing-a-scheduler.rst

```diff
@@ -97,10 +103,7 @@ to `enable the remote API`_.
     **Known Issues**
 
     - It is not yet possible to change the default affinity filter.
-    - If swarm can't create all the containers requested, Deis returns an error
-      and leaves some containers in the "created" state. Until this behavior is fixed,
-      be mindful of resource limitations on your cluster.
-    - App containers will not be rescheduled if deis-registry is unavailable.
+    - If swarm can't create all the containers requested during scale, deis rolls back the scale operation.
```

Comment:
I think now that you fixed `deis scale` to rollback if needed, this isn't a "Known Issue" any more, but the correct behavior. So maybe just delete the original bullet point but don't add this one in its place.